lib/colorado_lottery.rb

```diff
@@ -0,0 +1,58 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners,
+              :current_contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    if contestant.age < 18
+      false
+    elsif !contestant.game_interests.include?(game.name)
+      false
+    else
+      true
+    end
```

Comment:
this works! This could be done a bit more cleanly with some boolean logic:

```
 def interested_and_18?(contestant, game)
      contestant.age >= 18 && contestant.game_interests.include?(game.name)
  end
```